web/react/components/delete_post_modal.jsx

```diff
@@ -40,6 +40,14 @@ export default class DeletePostModal extends React.Component {
         ModalStore.removeModalListener(ActionTypes.TOGGLE_DELETE_POST_MODAL, this.handleToggle);
     }
 
+    componentDidUpdate() {
+        if (this.state.show) {
```

Comment:
1) Okay yup that makes sense, will do
2) Pretty sure it defaults to 0 (well, as close as it can get) from reading around, but you're right it's not clear.  I went with it because we don't give a timeout parameter to a similar function in `edit_post_modal.jsx` either.  I'll try changing it to `requestAnimationFrame`, hadn't used that before, thanks!